src/main/java/chess/domain/piece/Empty.java

```diff
@@ -0,0 +1,32 @@
+package chess.domain.piece;
+
+import chess.domain.Position;
+import java.util.List;
+
+public class Empty extends Piece {
+
+    private static final Empty empty = new Empty();
```

Comment:
```suggestion
    public static final Empty EMPTY = new Empty();
```
static 상수라면 대문자로 처리해주세요!
그리고 이런 경우라면 보통 of() 를 사용하지 않고 
public으로 열어두고 EMPTY를 그냥 사용하는게 좋을것 같아요!